ui/page/license_page.go

```diff
@@ -14,7 +14,7 @@ const LicensePageID = "License"
 
 const license = `ISC License
 
-Copyright (c) 2018-2021, Raedah Group
+Copyright (c) 2018-2022, Raedah Group
```

Comment:
we could look at automating the year in a future PR so we don't have to keep changing this
`year, _, _ := time.Now().Date()`
`fmt.Println(Copyright (c) 2018-%d, Raedah Group, year)`

not exact implementation, but something along those lines